public/assets/js/modules/profile.js

```diff
@@ -13,4 +13,53 @@ export function initializeProfile() {
             section.style.display = section.style.display === 'none' ? 'block' : 'none';
         }
     });
+   // Handle Image Preview and Validation for Profile Picture
```

Comment:
I think it would be more optimal to have a helper module for example  imageHandler.js
that would combine the shared logic for handling image validation and previews so it will be a reusable logic avoiding duplicating same logic in multible modules.

Reply:
Yes, this could work, but it would also require shared class name or Id in all form where image upload takes place

Reply:
FIXED